apps/OpenEnergyMonitor/myheatpump/myheatpump.js

```diff
@@ -493,12 +496,19 @@ $("#carnot_enable_prc").click(function(){
 $("#stats_when_running").click(function(){
     if ($("#stats_when_running")[0].checked) {
         $("#mean_when_running").show();
+        if (feeds["heatpump_dhw"]!=undefined) {
+            $("#stats_without_dhw").show();
+        }
     } else {
         $("#mean_when_running").hide();
+        $("#stats_without_dhw").hide();
     }
 
     powergraph_load();
-    powergraph_draw();
```

Comment:
Don't think the graph needs to be updated for the stats, so I removed this call.